core/sequencer/trillian_client.go

```diff
@@ -39,7 +39,7 @@ type trillianMap interface {
 	GetAndVerifyLatestMapRoot(ctx context.Context) (*tpb.SignedMapRoot, *types.MapRootV1, error)
 	SetLeavesAtRevision(ctx context.Context, rev int64, leaves []*tpb.MapLeaf, meta []byte) (*types.MapRootV1, error)
 	GetAndVerifyMapRootByRevision(ctx context.Context, rev int64) (*tpb.SignedMapRoot, *types.MapRootV1, error)
-	GetAndVerifyMapLeavesByRevision(ctx context.Context, rev int64, indexes [][]byte) ([]*tpb.MapLeaf, error)
+	GetMapLeavesByRevisionNoProof(ctx context.Context, rev int64, indexes [][]byte) ([]*tpb.MapLeaf, error)
```

Comment:
Maybe just `GetMapLeavesByRevision`?

Reply:
More generally, methods that verify proofs could be "GetAndVerifyBlah", while non-proof ones be just "GetBlah"?

Reply:
This mimicks the underlying API call.
In general, yes, the API names need a broader review